src/components/Register.js

```diff
@@ -64,6 +64,19 @@ class Register extends Component {
             // This gives you a Google Access Token. You can use it to access the Google API.
             let token = result.credential.accessToken;
 
+            let id = result.user.uid;
+            let firstName = result.user.displayName.split(' ')[0];
+            let lastName = result.user.displayName.split(' ')[1];
+            let email = result.user.email;
+            let pic = result.user.photoURL;
+            firebase.database().ref('users/' + id).set({
```

Comment:
Here you need to use update, otherwise Firebase will remove all the other fields you do not set here